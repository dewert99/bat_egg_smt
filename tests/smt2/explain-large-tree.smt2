(declare-sort ET)
(declare-fun f (ET ET) ET)
(declare-const al ET)
(define-const bl ET (f al al))
(define-const cl ET (f bl bl))
(define-const dl ET (f cl cl))
(declare-const ar ET)
(define-const br ET (f ar ar))
(define-const cr ET (f br br))
(define-const dr ET (f cr cr))
(check-sat-assuming ((= al ar) (not (= dl dr))))
(get-unsat-core)