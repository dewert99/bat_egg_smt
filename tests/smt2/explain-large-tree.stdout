unsat
((not (= dl dr)) (= al ar))
